### DISCUSSION TITLE
About Document Preparation

### DISCUSSION THREAD

**Discussion by 4d7568616d6d6564** [patrykandpatrick/vico#972] [Questions]
### Question

Hello, how can we use this amazing library in Java & Kotlin? I didn't find anything in the document. I asked ChatGPT and it didn't work. I want to create all the graphics in the featured images in Java. Can you prepare documents? Thank you!

### UI framework(s)

Views

**Comment by Gowsky** [comment]
Hello, @4d7568616d6d6564.

> Hello, how can we use this amazing library in **Java & Kotlin**?

> I want to create all the graphics in the featured images **in Java**.

Could you be more precise? Are you looking to use Vico with Kotlin, Java, or both?

**Reply by 4d7568616d6d6564** [reply]
Yes that's right I want to use it in my Java Android application

**Reply by Gowsky** [reply]
Please [see here](https://github.com/patrykandpatrick/vico/discussions/901#discussioncomment-10811865).
